unlock-app/src/components/interface/Drawer.tsx

```diff
@@ -51,10 +51,10 @@ export const Drawer = ({
               <div className="relative w-screen max-w-md">
                 <Transition.Child {...easeOutTransaction}>
                   <div className="absolute top-0 right-0 h-12 w-12 p-4">
-                    <Close size="16px" onClick={() => setIsOpen(false)} />
+                    <CloseIcon size="25px" onClick={() => setIsOpen(false)} />
                   </div>
                 </Transition.Child>
-                <div className="h-full flex flex-col py-6 bg-white shadow-xl overflow-y-scroll">
+                <div className="h-full flex flex-col py-6 bg-[#FFFDF8] shadow-xl overflow-y-scroll">
```

Comment:
Do we want to use a variable for that?

Reply:
added as brand color in tailwind config, for dashboard refactoring, we need it a lot